tasks.py

```diff
@@ -824,6 +824,26 @@ def webpack(clean=False, watch=False, dev=False):
 
 
 @task()
```

Comment:
I'm not exactly sure what you mean by this. Are you saying will the process be done when the Django App is started and therefore don't need a separate process? There does need to be something to call when js files are changed (like invoke assets for the osf) but I'm not how to do this exactly.